source/boot_security.c

```diff
@@ -1,21 +1,35 @@
 #include "boot_security.h"
 #include "aes.h"
```

Comment:
I asked for the file header as explained in https://github.com/OpenIoTAcademy/psa-boot-beta/wiki/File-Comments-in-Doxygen-Style

Reply:
a